src/vulkan-renderer/wrapper/mesh_buffer.cpp

```diff
@@ -90,10 +106,14 @@ MeshBuffer::MeshBuffer(const wrapper::Device &device, VkQueue data_transfer_queu
     staging_buffer_for_vertices.upload_data_to_gpu(m_vertex_buffer);
 }
 
-MeshBuffer::MeshBuffer(MeshBuffer &&other) noexcept
-    : m_name(std::move(other.m_name)), m_vertex_buffer(std::move(other.m_vertex_buffer)),
-      m_index_buffer(std::move(other.m_index_buffer)), m_number_of_vertices(other.m_number_of_vertices),
-      m_number_of_indices(other.m_number_of_indices), m_device(other.m_device) {}
+MeshBuffer::MeshBuffer(const wrapper::Device &device, const VkQueue data_transfer_queue,
```

Comment:
`data_transfer_queue` and `data_transfer_queue_family_index` are never used.

Reply:
Not just there: it's a general issue in the code.
I will deal with this in another PR: https://github.com/inexorgame/vulkan-renderer/issues/238.